doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,136 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit to function as a bluetooth speaker
+=====================================================================
+
+For bluetooth speaker setup you should use the "gateway" image and
+prepare couple of configuration files. This configuration has been
+tested to work with Intel Joule development board which has bluetooth
+radio chip integrated. Following 5 steps should make it:
+
+1. Start pulseaudio in the boot
+2. Add D-Bus configuration so that pulseaudio can communicate with BlueZ
+3. Configure BlueZ to enable A2DP sink
+4. Unblock bluetooth at boot with Connman
+5. Pair and connect your source device
+
+Start pulseaudio in the boot
+============================
+
+The way to start pulseuadio in the boot is to use systemd service
+file. In this example service file we start pulseaudio in system mode
+and don't let the daemon exit when in idle (which is normal
+pulseuaudio behavior). Pulseaudio will just go to "sleep"::
+
+  pulseaudio
+  [Unit]
+  Description=Pulse Audio
+
+  [Service]
+  Type=simple
+  ExecStart=/usr/bin/pulseaudio --system --disallow-exit --disable-shm --exit-idle-time=-1
+
+  [Install]
+  WantedBy=multi-user.target
+
+Copy the pulseaudio.service file to::
+
+  /etc/systemd/system/
+
+and enable it by::
+
+  systemctl enable pulseaudio.service
+
+For this particular use case we are starting pulseaudio in system mode
+and including bluetooth modules to the system.pa file. Remember that
+starting pulseaudio in system mode is not the recommended way and you
+should be aware of the possible consequences.
```

Comment:
This kind of begs the question: why don't we provide instructions on how to start PulseAudio the recommended way?